src/plugins/intel_cpu/tests/functional/single_layer_tests/nms_rotated.cpp

```diff
@@ -0,0 +1,338 @@
+        targetDevice = utils::DEVICE_CPU;
+
+        const auto& params          = this->GetParam();
+        const auto& in_shapes       = std::get<0>(params);
+        const auto& boxes_prc       = std::get<1>(params);
+        const auto& max_boxes_prc   = std::get<2>(params);
+        const auto& thresholds_prc  = std::get<3>(params);
+        const auto& out_prc         = std::get<4>(params);
+        m_max_out_boxes_per_class   = std::get<5>(params);
+        m_iou_threshold             = std::get<6>(params);
+        m_score_threshold           = std::get<7>(params);
+        const auto& sort_descending = std::get<8>(params);
+        const auto& clockwise       = std::get<9>(params);
+        const auto& is_0_in_const   = std::get<10>(params);
+        const auto& is_1_in_const   = std::get<11>(params);
+        const auto& is_2_in_const   = std::get<12>(params);
+        const auto& is_3_in_const   = std::get<13>(params);
+        const auto& is_4_in_const   = std::get<14>(params);
+        const auto& cpu_params      = std::get<15>(params);
+        configuration               = std::get<16>(params);
+
+        std::tie(inFmts, outFmts, priority, selectedType) = cpu_params;
+
+        std::vector<InputShape> actual_shapes;
+        ov::ParameterVector in_params;
+        std::vector<std::shared_ptr<ov::Node>> inputs;
+        const auto in_shape_1d = InputShape{{1}, {{1}}};
+
+#define CONST_CASE(P, S, H, L)                                                                                                 \
+        case P:                                                                                                                \
+            inputs.push_back(ngraph::builder::makeConstant(P, S, std::vector<ov::element_type_traits<P>::value_type>{}, true,  \
+                             ov::element_type_traits<P>::value_type(H), ov::element_type_traits<P>::value_type(L)));           \
+            break;
+
+#define CREATE_INPUT(C, P, S, N, H, L)                                                                                         \
+        if (C) {                                                                                                               \
+            switch (P) {                                                                                                       \
+                CONST_CASE(ElementType::f32,  S.second[0], H, L)                                                               \
+                CONST_CASE(ElementType::f16,  S.second[0], H, L)                                                               \
+                CONST_CASE(ElementType::bf16, S.second[0], H, L)                                                               \
+                CONST_CASE(ElementType::i32,  S.second[0], H, L)                                                               \
+                CONST_CASE(ElementType::i64,  S.second[0], H, L)                                                               \
```

Comment:
Please add `default` section here: this will most likely fix a compilation error in CI

Reply:
Done